packages/lesswrong/components/sequences/ChaptersItem.jsx

```diff
@@ -19,14 +35,14 @@ class ChaptersItem extends Component {
   }
 
   renderTitleComponent = (chapter, canEdit) => <div>
-    {chapter.subtitle ?   <div className="chapters-item-subtitle">
+    {chapter.subtitle ? <div className={this.props.classes.subtitle}>
```

Comment:
I won't hold off approval for this change, but I think whenever you find yourself typing "this.props.propName" it'd be nicer if you went ahead and did the "const { propName } = this.props" thing